Extensions/xkit_patches.js

```diff
@@ -269,6 +269,329 @@ XKit.extensions.xkit_patches = new Object({
 				});
 				return posts;
 			};
+
+			/**
+			 * @param {JQuery} $post - Post element
+			 * @return {Object} Interface Post Object or {error: true}
+			 */
+			XKit.interface.parse_post = function($post) {
```

Comment:
I think renaming `interface.post` makes this pull request a lot harder to review then it needs to be, and it has the potential to break dozens of non-centralized XKit userscripts. It also leaves many confusingly-named functions in its place, like `find_post` and `get_posts`.

Reply:
a very valid point, but i honestly cant think of what else to name the new thing - if you have even a half-decent suggestion i can leave `interface.post` as it was